src/gui/org/deidentifier/arx/gui/view/impl/define/ViewAttributeList.java

```diff
@@ -269,6 +278,31 @@ private void create(final Composite parent) {
                     menu.setLocation(table.toDisplay(e.x, e.y));
                     menu.setVisible(true);
                 }
+                
+                Point pt = new Point(e.x, e.y);
+                int index = table.getTopIndex();
+                while (index < table.getItemCount()) {
+                    TableItem item = table.getItem(index);
+                    for (int i = 0; i < 5; i++) {
+                        Rectangle rect = item.getBounds(i);
+                        if (rect.contains(pt)) {
+                            if (i == 4) {
+                                String responseVariable = model.getInputConfig().getInput().getHandle().getAttributeName(index);
+                                // If already contained, remove
+                                boolean isResponseVariable = true;
+                                if(model.getInputDefinition().isResponseVariable(responseVariable)) {
+                                    isResponseVariable = false;
+                                }
+                                model.getInputDefinition().setResponseVariable(responseVariable, isResponseVariable);
+                                updateEntries();
```

Comment:
It would be much better, to directly update the item here instead of calling "updateEntries". Why not just item.setImage(4, IMAGE)?